git-and-github-for-policital-organizing.md

```diff
@@ -0,0 +1,26 @@
+> tech antifa curriculum
+
+# git and GitHub for Politicial Organizing
+
+### Prerequisite Knowledge:
+
+- Familiarity with navigating a computer
+- Experience with a text editor (any)
+
+### Equipment:
+
+- Laptop (any OS with a terminal and browser)
```

Comment:
There are also several git [GUIs](https://git-scm.com/downloads/guis). I've never used any, but in case anybody is intimidated by the terminal